renku/cli/workflow.py

```diff
@@ -599,3 +607,58 @@ def outputs(ctx, paths):
             p not in output_paths and all(Path(o) not in Path(p).parents for o in output_paths) for p in paths
         ):
             ctx.exit(1)
+
+
+@workflow.command()
+@click.option(
+    "provider",
+    "-p",
+    "--provider",
+    default="cwltool",
+    show_default=True,
+    type=click.Choice(available_workflow_providers(), case_sensitive=False),
+    help="The workflow engine to use.",
+)
+@click.option("config", "-c", "--config", metavar="<config file>", help="YAML file containing config for the provider.")
+@click.option(
+    "set_params",
+    "-s",
+    "--set",
+    multiple=True,
+    metavar="<parameter>=<value>",
```

Comment:
How does this work with `CompositePlanValueResolver`? In there we look for a `parameters` or a `steps` keys. It's also probably good to add a test which uses this flag.